packages/chaire-lib-backend/src/models/db/__tests__/users.db.test.ts

```diff
@@ -51,14 +62,17 @@ afterAll(async() => {
 });
 
 each([
-    [{ first_name: user.first_name }, user ],
+    [{ first_name: user.first_name }, user],
     [{}, undefined],
     [{ first_name: 'other'}, undefined],
     [{ first_name: user2.first_name, username: user2.username }, user2],
     [{ usernameOrEmail: user2.email }, user2],
     [{ usernameOrEmail: user2.username }, user2],
     [{ password_reset_token: user2.password_reset_token }, user2],
     [{ username: user2.username, email: 'arbitrary' }, undefined],
+    [{ usernameOrEmail: user2.email.toUpperCase() }, undefined],
+    [{ usernameOrEmail: userCaps.email }, userCaps],
+    [{ username: user.username, email: userCaps.email}, undefined]
```

Comment:
Here too, userCaps and user both have the same email, so it should return the user object